fe/fe-core/src/main/java/com/starrocks/scheduler/PartitionBasedMvRefreshProcessor.java

```diff
@@ -1572,9 +1572,7 @@ private boolean checkBaseTablePartitionChange() {
                     }
                 } else if (ConnectorPartitionTraits.isSupported(snapshotTable.getType())) {
                     if (snapshotTable.isUnPartitioned()) {
-                        if (!table.isUnPartitioned()) {
-                            return true;
-                        }
+                        return false;
```

Comment:
1609 lines have a default return false, Would it be better to use the following code :
```
if (!snapshotTable.isUnPartitioned()) {
```